teacher.rb

```diff
@@ -0,0 +1,11 @@
+# teacher with there specialization and servies
+class Teacher < Person
```

Comment:
- [ ] For this inheritance to work you need to import the Person class into this file. Kindly do that.

  ```
  require_relative './person'
  ```